lib/vagrant-openshift/version.rb

```diff
@@ -16,6 +16,6 @@
 
 module Vagrant
   module Openshift
-    VERSION = "1.0.19"
+    VERSION = "1.0.20"
```

Comment:
In general you don't need to bump the version.  This will happen when published.